vyper/compiler/output.py

```diff
@@ -300,9 +300,13 @@ def _build_opcodes(bytecode: bytes) -> str:
 
     while bytecode_sequence:
         op = bytecode_sequence.popleft()
-        opcode_output.append(opcode_map[op])
+        opcode_output.append(opcode_map.get(op, f"VERBATIM_{hex(op)}"))
         if "PUSH" in opcode_output[-1] and opcode_output[-1] != "PUSH0":
             push_len = int(opcode_map[op][4:])
+            # we can have push_len > len(bytecode_sequence) when there is data
+            # (instead of code) at end of contract
+            # CMC 2023-07-13 maybe just strip known data segments?
+            push_len = min(push_len, len(bytecode_sequence))
```

Comment:
@charles-cooper Is it possible for `len(bytecode_sequence)` to be `< push_len` here and if that is the case, would that cause isses?

Reply:
yes, but it shouldn't cause issues, this is just a way of handling disassembly of the data section